cmd/maestro/environments/types.go

```diff
@@ -22,6 +23,7 @@ const (
 type Env struct {
 	Name          string
 	Services      Services
+	Dispatcher    dispatcher.Dispatcher
```

Comment:
what is a "Dispatcher"?  is there a grouping of things we're missing on this struct? 

This seems like the wrong place, too, as there are no other _single things_ on this struct.

Reply:
nm, i see the Dispatcher config struct.